### ISSUE TITLE
Error during installation via npm with version 2.1.0

### ISSUE THREAD

**Issue by AccsoSG** [anvilco/spectaql#837]
I get an error message when installing version 2.1.0, but 2.0.9 works.

````
PS C:\xxx\xxx\xxx> npm i spectaql
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm WARN deprecated opn@6.0.0: The package has been renamed to `open`
npm ERR! code 1
npm ERR! path C:\xxx\xxx\xxx\node_modules\spectaql
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c npm run build:vendor
npm ERR! > spectaql@2.1.0 build:vendor
npm ERR! > node bin/build-vendor.mjs
npm ERR!
npm ERR! Building managed vendor dependency: grunt-compile-handlebars
npm ERR! npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
npm ERR! Der Befehl "rm" ist entweder falsch geschrieben oder
npm ERR! konnte nicht gefunden werden.
npm ERR! node:internal/errors:841
npm ERR!   const err = new Error(message);
npm ERR!               ^
npm ERR!
npm ERR! Error: Command failed: rm C:\xxx\xxx\xxx\node_modules\spectaql\vendor\grunt-compile-handlebars-1.0.0.tgz
npm ERR! Der Befehl "rm" ist entweder falsch geschrieben oder
npm ERR! konnte nicht gefunden werden.
npm ERR!
npm ERR!     at checkExecSyncError (node:child_process:828:11)
npm ERR!     at execSync (node:child_process:899:15)
npm ERR!     at file:///C:/xxx/xxx/xxx/node_modules/spectaql/bin/build-vendor.mjs:118:11 {
npm ERR!   status: 1,
npm ERR!   signal: null,
npm ERR!   output: [
npm ERR!     null,
npm ERR!     Buffer(0) [Uint8Array] [],
npm ERR!     Buffer(85) [Uint8Array] [
npm ERR!        68, 101, 114,  32,  66, 101, 102, 101, 104, 108,  32,  34,
npm ERR!       114, 109,  34,  32, 105, 115, 116,  32, 101, 110, 116, 119,
npm ERR!       101, 100, 101, 114,  32, 102,  97, 108, 115,  99, 104,  32,
npm ERR!       103, 101, 115,  99, 104, 114, 105, 101,  98, 101, 110,  32,
npm ERR!       111, 100, 101, 114,  13,  10, 107, 111, 110, 110, 116, 101,
npm ERR!        32, 110, 105,  99, 104, 116,  32, 103, 101, 102, 117, 110,
npm ERR!        10
npm ERR!     ]
npm ERR!   ],
npm ERR!   pid: 17492,
npm ERR!   stdout: Buffer(0) [Uint8Array] [],
npm ERR!   stderr: Buffer(85) [Uint8Array] [
npm ERR!      68, 101, 114,  32,  66, 101, 102, 101, 104, 108,  32,  34,
npm ERR!     114, 109,  34,  32, 105, 115, 116,  32, 101, 110, 116, 119,
npm ERR!     101, 100, 101, 114,  32, 102,  97, 108, 115,  99, 104,  32,
npm ERR!     103, 101, 115,  99, 104, 114, 105, 101,  98, 101, 110,  32,
npm ERR!     111, 100, 101, 114,  13,  10, 107, 111, 110, 110, 116, 101,
npm ERR!      32, 110, 105,  99, 104, 116,  32, 103, 101, 102, 117, 110,
npm ERR!     100, 101, 110,  32, 119, 101, 114, 100, 101, 110,  46,  13,
npm ERR!      10
npm ERR!   ]
npm ERR! }

npm ERR! A complete log of this run can be found in: C:\Users\xxx\AppData\Local\npm-cache\_logs\2023-06-15T14_44_56_071Z-debug-0.log
````

**Comment by newhouse** [comment]
OK @AccsoSG this is fixed in `2.1.1`. Please let me know if you are good now.

**Comment by AccsoSG** [comment]
That was fast!
Yes it works with 2.1.1, thank you very much

**Comment by gerhard** [comment]
Hi, we have hit the same issue with `v2.1.1`, in the context of https://github.com/dagger/dagger/pull/5395:

```js
[4/4] 🔨  Building fresh packages...
error /Users/gerhard/github.com/dagger/dagger/website/node_modules/spectaql: Command failed.
Exit code: 1
Command: npm run build:vendor
Arguments:
Directory: /Users/gerhard/github.com/dagger/dagger/website/node_modules/spectaql
Output:
> spectaql@2.1.1 build:vendor
> node bin/build-vendor.mjs

Building managed vendor dependency: grunt-compile-handlebars
node:internal/errors:867
  const err = new Error(message);
              ^

Error: Command failed: npm pack --pack-destination /Users/gerhard/github.com/dagger/dagger/website/node_modules/spectaql/vendor
    at checkExecSyncError (node:child_process:885:11)
    at execSync (node:child_process:957:15)
    at file:///Users/gerhard/github.com/dagger/dagger/website/node_modules/spectaql/bin/build-vendor.mjs:92:29 {
  status: 1,
  signal: null,
  output: [ null, Buffer(0) [Uint8Array] [], null ],
  pid: 48119,
  stdout: Buffer(0) [Uint8Array] [],
  stderr: null
}

Node.js v18.15.0
npm notice
npm notice New minor version of npm available! 9.5.0 -> 9.7.2
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v9.7.2>
npm notice Run `npm install -g npm@9.7.2` to update!
npm notice
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

We will close the dependabot auto-bump and maybe try it out manually later. If not, dependabot will try again next time.

Thank you!
